### PR TITLE
Split register section in tabs

### PR DESCRIPTION
**Pull request by reivaj05**
Splits Register section in tabs.

This was done by @bexsoft, i'm just creating a new PR with his changes since his other PR where this is implemented is blocked for the moment: https://github.com/minio/console/pull/2168

Screen Shot 2022-07-18 at 14 45 42
Screen Shot 2022-07-18 at 14 45 55

### CODE REVIEW

**Review by cniackz — APPROVED** [review]
LGTM

File: portal-ui/src/screens/Console/Support/Register.tsx [hunk 1]
```diff
@@ -913,46 +943,7 @@ const Register = ({ classes }: IRegister) => {
...
-                  fetchSubnetRegToken();
-                  setOnlineActivation(!onlineActivation);
-                }}
-              >
-                Offline Activation.
-              </Link>
-            </Box>
-          </Box>
-        </React.Fragment>
+        </Fragment>
```
Comment (jinapurapu) [inline portal-ui/src/screens/Console/Support/Register.tsx:946]: This field is always disabled (at 912)
```suggestion
        {displaySubnetProxy && (
                  <InputBoxWrapper
                    disabled
                    id="subnetProxy"
                    name="subnetProxy"
                    placeholder=""
                    onChange={() => {}}
                    label=""
                    value={proxyConfigurationCommand}
                    overlayIcon={<CopyIcon />}
                    extraInputProps={{
                      readOnly: true,
                    }}
                    overlayAction={() =>
                      navigator.clipboard.writeText(proxyConfigurationCommand)
                    }
                  />
                )}
```

**Review by jinapurapu — CHANGES_REQUESTED** [review]
The proxy input field is always disabled

File: portal-ui/src/screens/Console/Support/Register.tsx (hunk 1 above)
Comment (reivaj05) [reply portal-ui/src/screens/Console/Support/Register.tsx:946]: That's the expected behavior, we can only copy the text inside the input

**Review by dvaldivia — APPROVED** [review]

**Pull request merged**